internal/testcontext/compile.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testcontext
+
+import (
+	"os/exec"
+	"path"
+)
+
+// Compile compiles the specified package and returns the executable name.
+func (ctx *Context) Compile(pkg string) string {
+	ctx.test.Helper()
+
+	exe := ctx.File("build", path.Base(pkg)+".exe")
```

Comment:
Why a `.exe` extension? Does this not work on windows otherwise or something?

Reply:
Yup, Windows.